cal/api/views.py

```diff
@@ -224,14 +224,6 @@ def get_serializer_context(self):
 
     def get_queryset(self):
         qs = Tag.objects.filter(user=self.request.user)
-        start = self.request.query_params.get('start')
-        end = self.request.query_params.get('end')
-
-        if start:
-            qs = qs.filter(start__gte=start)
-        if end:
-            qs = qs.filter(end__lte=end)
```

Comment:
This breaks it - a `Tag` doesn't have a start parameter so it causes a 500 error